all-the-stars-js/js/achievements.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
Can you go back to using the constants.js file vs. breaking them out?  The constants.js file is downloadable from the play console, so we encourage that to minimize the amount of copy and paste during configuration.